crates/anvil/core/src/eth/transaction/optimism.rs

```diff
@@ -229,6 +229,14 @@ impl Transaction for DepositTransactionRequest {
     fn value(&self) -> U256 {
         self.value
     }
+
+    fn encode_for_signing(&self, out: &mut dyn alloy_rlp::BufMut) {
+        self.encode_for_signing(out)
+    }
+
+    fn payload_len_for_signature(&self) -> usize {
+        self.payload_len_for_signature()
+    }
```

Comment:
unrelated, comes from the alloy bump, see https://github.com/alloy-rs/alloy/pull/161